### PR TITLE
Ras Bid Adapter: update ras adapter and rename it to ringieraxelspringer

### PR DESCRIPTION
**Pull request by wsusrasp**
## Type of change
- Renamed ras bid adapter to ringieraxelspringer
- Minor tweaks and test updates

## Description of change

Due to internal reasons we need to change the bidder code, the `ras` is now deprecated.

### CODE REVIEW

**Comment by jsnellbaker** [comment]
@wsusrasp If you haven't already, please submit a prebid Docs PR as well to reflect these changes:
https://github.com/prebid/prebid.github.io/pulls

File: modules/ringieraxelspringerAnalyticsAdapter.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import adapter from '../libraries/analyticsAdapter/AnalyticsAdapter.js';
+import adapterManager from '../src/adapterManager.js';
+
+var ringieraxelspringerAdapter = adapter({
+  global: 'ringieraxelspringerPrebidAnalytics',
+  handler: 'on',
+  analyticsType: 'bundle'
+});
+
+adapterManager.registerAnalyticsAdapter({
```
Comment (patmmccann) [inline modules/ringieraxelspringerAnalyticsAdapter.js:10]: am I out of date, don't you need an endpoint or something? How can an analytics adapter be this compact?

Comment (wsusrasp) [reply modules/ringieraxelspringerAnalyticsAdapter.js]: I'll need to check in a moment, but this is what we were using up until now. I'll move these files to new PR as requested for now.

**Review by patmmccann — APPROVED** [review]

**Comment by wsusrasp** [comment]
Here is related docs request: https://github.com/prebid/prebid.github.io/pull/5383
Thank you for quick review

**Pull request merged**